tools/rna_tools/rnaformer/.shed.yml

```diff
@@ -0,0 +1,17 @@
+name: rnaformer
+owner: ivelet
```

Comment:
```suggestion
owner: rnateam
```

Reply:
ping

Reply:
Thank you @bgruening but it appears that there are still serious issues with the automated tests which I can't understand or reproduce locally (the same planemo tests used here pass when done locally) even when testing it by removing all the tool's functionality. I have changed the PR back to draft because of this since it will fail the tests anyway.

Reply:
Update: This was fixed in the latest commit and all the tests now pass.

Reply:
mh, what are the problem?